lib/platform.py

```diff
@@ -61,6 +62,7 @@
   AVC_ENCODE_PLATFORMS          = [                     "BDW",        "SKL", "APL", "KBL",        "CFL", "WHL", "ICL"]
   AVC_ENCODE_LP_PLATFORMS       = [                                   "SKL", "APL", "KBL",        "CFL", "WHL", "ICL"]
   HEVC_DECODE_8BIT_PLATFORMS    = [                                   "SKL", "APL", "KBL",        "CFL", "WHL", "ICL"]
+  HEVC_DECODE_8BIT_8K_PLATFORMS = [                                                                             "ICL"]
```

Comment:
AFAICT, KBL appears to support up to 8K, too, on iHD.  Possibly other platforms support 8K as well.  I am able to decode 8K hevc on my KBL with iHD.  Can you verify supported platforms?